charts/kong/README.md

```diff
@@ -670,6 +670,8 @@ or `ingress` sections, as it is used only for stream listens.
 |-----------------------------------|-------------------------------------------------------------------------------------------|--------------------------|
 | SVC.enabled                       | Create Service resource for SVC (admin, proxy, manager, etc.)                             |                          |
 | SVC.http.enabled                  | Enables http on the service                                                               |                          |
+| SVC.http.serviceEnabled           | Enables http on the proxy service (to ensure backward compatibility its gets overridden by SVC.http.enabled if unset)                                                               |                          |
+| SVC.http.containerEnabled         | Expose http port on container (to ensure backward compatibility its overridden by SVC.http.enabled if unset)                                                            |                          |
```

Comment:
If we don't go with the suggestion to overload `servicePort` and `containerPort` elsewhere, this should delete `(to ensure backward compatibility its overridden by SVC.http.enabled if unset)` and just list a default `true` value.

Reply:
will be updating the logic considering your suggestion in this comment https://github.com/Kong/charts/pull/1021#discussion_r1509554696 accordingly will update the README.md

Reply:
updated the code with special handling of servicePort=0 to handle the use-case instead of adding extra knobs